### DISCUSSION TITLE
Big Sur and sshfs

### DISCUSSION THREAD

**Discussion by kavehv** [Homebrew/discussions#1343] [Casks]
I can't seem to get sshfs working in big sur.  Every time I go to install it I get:

> Error: sshfs has been disabled because it requires FUSE!
I've installed macFUSE (the latest version) but brew seems to think it needs an older osxFuse instead.  Has anyone looked at this in big sur?

**Comment by carlocab** [comment]
See https://github.com/Homebrew/homebrew-core/issues/75656.
